mmaction/models/backbones/resnet.py

```diff
@@ -432,13 +432,15 @@ def _load_conv_params(conv, state_dict_tv, module_name_tv,
         """
 
         weight_tv_name = module_name_tv + '.weight'
-        conv.weight.data.copy_(state_dict_tv[weight_tv_name])
-        loaded_param_names.append(weight_tv_name)
+        if conv.weight.data.shape == state_dict_tv[weight_tv_name].shape:
```

Comment:
do we need to issue some warning if the shapes are mismatched?

Reply:
I think no warning is OK, since: 
1. The shape mismatch may only happen when in_channels != 3. In this case, the shape mismatch is not **unexpected**. Users assume that pre-train weights are for other modules (except conv1).
2. There is already a warning message in `_load_torchvision_checkpoint`, which indicates that parameters of `conv1` and `bn1` are not loaded.

Reply:
Its okay if pytorch could warn the user. As long as there are warnings, I'm ok with the change.